app/generator/content/part2.js

```diff
@@ -0,0 +1,120 @@
+  let i = 1
+  funding.forEach(x => {
+    if (x.reductions) {
+      x.reductions.forEach(y => {
+        if (!reductions.find(z => z.reason === y.reason)) {
+          reductions.push({
+            id: i,
+            reason: y.reason
+          })
+          i++
+        }
+      })
+    }
+  })
+  return reductions
+}
+
+const getTable = (funding, reductions) => {
+  return {
+    layout: {
+      hLineStyle: () => 'solid',
+      vLineStyle: () => 'solid'
+    },
+    style: 'table',
+    table: {
+      headerRows: 1,
+      widths: ['*', '*', '*', '*', '*', '*', '*', '*'],
+      body: [
+        [
+          { text: 'Standard', style: 'tableHeader' },
+          { text: 'Level', style: 'tableHeader' },
+          { text: 'Rate', style: 'tableHeader' },
+          { text: 'Land area (ha)', style: 'tableHeader' },
+          { text: 'Annual value', style: 'tableHeader' },
+          { text: 'Quarterly value', style: 'tableHeader' },
+          { text: 'Quarterly reduction', style: 'tableHeader' },
+          { text: 'Quarterly reduction', style: 'tableHeader' }
+        ],
+        ...mapFundingRows(funding.filter(x => x.name !== 'Total'), reductions),
+        mapTotalRow(funding.find(x => x.name === 'Total'))
+      ]
+    }
+  }
+}
+
+const mapFundingRows = (funding, reductions) => {
+  return funding.map(x => ([
+    x.name,
+    x.level,
+    { text: x.rate, style: 'tableNumber' },
+    { text: x.area, style: 'tableNumber' },
+    { text: `£${x.annualValue}`, style: 'tableNumber' },
+    { text: `£${x.quarterlyValue}`, style: 'tableNumber' },
+    { text: `£${x.quarterlyReduction}${mapReductionIds(x.reductions, reductions)}`, style: 'tableNumber' },
+    { text: `£${x.quarterlyPayment}`, style: 'tableNumber' }
+  ]))
+}
+
+const mapTotalRow = (row) => {
+  return [
+    { text: row.name, bold: true },
+    { text: row.level, bold: true },
+    { text: row.rate, style: 'tableNumber', bold: true },
+    { text: row.area, style: 'tableNumber', bold: true },
+    { text: `£${row.annualValue}`, style: 'tableNumber', bold: true },
+    { text: `£${row.quarterlyValue}`, style: 'tableNumber', bold: true },
+    { text: `£${row.quarterlyReduction}`, style: 'tableNumber', bold: true },
+    { text: `£${row.quarterlyPayment}`, style: 'tableNumber', bold: true }
+  ]
+}
+
+const mapReductionIds = (fundReductions, reductions) => {
+  if (!fundReductions.length) {
+    return ''
+  }
+  return `\n${(fundReductions.map(x => {
+    const reduction = reductions.find(y => y.reason === x.reason)
+    return `\n(${reduction.id})`
+  })).join('')}`
+}
+
+const getReductions = (reductions) => {
+  if (!reductions.length) {
+    return []
+  }
+  return [
+    { text: 'Reason for reductions', style: 'header3' },
+    ...getReductionList(reductions)
+  ]
+}
+
+const getReductionList = (reductions) => {
+  const list = []
+  reductions.forEach(x => {
+    list.push(`(${x.id}) ${x.reason}`)
+  })
+  return list
+}
+
+module.exports = part2
```

Comment:
got some discrepancies using the `mock-statement-data`

<img width="951" alt="Screenshot 2022-08-08 at 15 31 56" src="https://user-images.githubusercontent.com/56440327/183442753-4d1b03be-d9a4-47de-906e-60057cc8f05d.png">

Quarterly payment for improved grasslands should be: 105-50 = 45 but is 92.50 (which is 105-(50/4) as if the qtr calc hadn't been done)

also, the total for payments (inc'ing the erroneous 92.50) is 100 quid more than it should be: actually 242.15